doc/api.md

```diff
@@ -225,6 +226,36 @@ will be presented in the live content store. Uses
   - All published editions that link to this item (directly or through a
     recursive chain of links) will be updated in the live content store.
 
+## `POST /v2/content/:content_id/publish`
+
+Used to set a live edition to a published state, used to remove unpublishing
+or to re-send published data downstream. Uses
+[optimistic-locking](#optimistic-locking-previous_version).
+
+### Path parameters
+
+- [`content_id`](model.md#content_id)
+  - Identifies the document which has an edition to republish.
+
+### JSON attributes
+
+- `locale` *(optional, default: "en")*
+  - Accepts: An available locale from the [Rails I18n gem][i18n-gem]
+  - Specifies the locale of the document.
+- `previous_version` *(optional, recommended)*
```

Comment:
I realise it goes against the other endpoints, but since this is a new endpoint which will presumably only be used bye one app at the moment could we consider making this a required field?

It feels like it should always have been a required field to me.

Reply:
The problem is that no-one uses it as far as I'm aware, we're not using it on Content Publisher. I think it makes more sense if Publishing API is the source of truth but when you're syncing state to Publishing API then I guess we care more that we're up to date.